is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
+      if (elementToCompare === currentElement) {
+        isAllUnique = false;
```

Comment:
There could be a return on this line to be more efficient, since the rest of the array doesn't need to be checked if one of them is not unique.

Reply:
Which line are you referring to, @ktvoort123? It's my understanding that using ```.forEach`` will not allow you to terminate early; it executes once for each item. (Perhaps this introduces the question of redesigning the expression to optimize for efficiency using a different pattern or data structure?)